harness/features/initialize.local.test.ts

```diff
@@ -356,10 +339,10 @@ describe('Initialize Tests - Local', () => {
         const testClient = new LocalTestClient(sdkName)
         scope
             .get(
-                `/client/${testClient.clientId}/config/v1/server/${testClient.sdkKey}.json`,
+                testClient.getValidConfigPath(),
             )
             .times(1)
-            .reply(200, config, {
+            .reply(200, testClient.getValidConfig, {
```

Comment:
```suggestion
            .reply(200, testClient.getValidConfig(), {
```

Reply:
would probably just be easier to pull out config at the start of this test into a `config` const rather than keep calling `testClient.getValidConfig()`

Reply:
I kinda wanted to just make it part of the constructor and have a param to override and use a custom pull

Reply:
also you are still not calling the function here, `()` are missing.

Reply:
ah - just didn't push it locally